src/errors.js

```diff
@@ -82,9 +82,15 @@ export function onlyErrorsWithCode(
   return (error) => {
     let throwError = true;
 
-    if (Array.isArray(codeWanted) && codeWanted.indexOf(error.code) !== -1) {
-      throwError = false;
-    } else if (error.code === codeWanted) {
+    if (Array.isArray(codeWanted)) {
+      if (codeWanted.indexOf(error.code) !== -1) {
+        throwError = false;
+      }
+      if (codeWanted.indexOf(error.errno) !== -1) {
```

Comment:
NodeJS is our target platform with 4.0 being the minimum. I just checked and sadly 4.0 doesn't have this function.